### PR TITLE
Use system fonts and remove broken on android shaders

### PR DESCRIPTION
**Pull request by MoNTE48**
It works fine in MultiCraft many months.
These fonts are required in ANY version and ROMs. They used Google Apps too.

### CODE REVIEW

**Comment by sofar** [comment]
The commit message is wrong. The shaders are not broken, it's Android clients that are broken and can not display them.

I'm OK with removing them from Android installations, but why can't they be set to disabled instead?

**Comment by MoNTE48** [comment]
@sofar, what for? For 3 years of no fixed shader on android port. If you want to disable it instead of the comment line - make your pull. I just copied the code from MultiCraft

**Comment by sofar** [comment]
@MoNTE48 because that's how the Minetest team works. Take a look a PR's that I've submitted, and see that people asked me to change them many times, and I did, and then it gets merged.

You're guilty of "dump and run" practices: You claim that you care, but you're unwilling to make small and logical adjustments that will help everyone, including yourself.

Now, if you don't want to modify it, that's entirely fine, and you're free to do so. Perhaps my comment is wrong, or perhaps you don't care. But don't complain if your PR's are not getting merged fast enough if you're unwilling to compromise and come to some sort of constructive collaboration.

No hard feelings, I'm trying to *help* you get these changes merged, not sabotage them.

**Comment by MoNTE48** [comment]
@sofae, I realized, I will update the PR when I'll be home. I do not like to work in a team, I'm sorry :) Your advice is very important to me and I thank you.

**Comment by est31** [comment]
Yes, many if not most open source projects work like @sofar described.

**Comment by MoNTE48** [comment]
@sofar, I have no idea whether it is possible to do so. Check, please. I can not check it out, now I have only windows tablet.

File: build/android/Makefile [hunk 1]
```diff
@@ -731,10 +732,12 @@ assets : $(ASSETS_TIMESTAMP)
 	cp ${ROOT}/../../README.txt ${ROOT}/assets/Minetest;                       \
 	cp -r ${ROOT}/../../builtin ${ROOT}/assets/Minetest;                       \
 	mkdir ${ROOT}/assets/Minetest/client;                                      \
+ifeq ($(ENABLE_SHADERS),1)                                                     \
 	cp -r ${ROOT}/../../client/shaders ${ROOT}/assets/Minetest/client;         \
+endif                                                                          \
 	cp ${ROOT}/../../doc/lgpl-2.1.txt ${ROOT}/assets/Minetest/LICENSE.txt;     \
 	mkdir ${ROOT}/assets/Minetest/fonts;                                       \
-	cp -r ${ROOT}/../../fonts/*.ttf ${ROOT}/assets/Minetest/fonts/;            \
+	cp -r ${ROOT}/../../fonts/liberationsans ${ROOT}/assets/Minetest/fonts/;   \
```
Comment (ShadowNinja) [inline build/android/Makefile]: Why is this line still here?  There's no directory named `liberationsans`.

Comment (MoNTE48) [inline build/android/Makefile]: Oops, I not add ttf to file name.

Comment (ShadowNinja) [inline build/android/Makefile]: Why copy it at all though?  Is it still used?

Comment (MoNTE48) [inline build/android/Makefile]: Yes. Minetest use 3 fonts, android contains only 2 of them

**Comment by rubenwardy** [comment]
lgtm, apart from ShadowNinja's comment

**Comment by rubenwardy** [comment]
won't this be fixed by https://github.com/minetest/minetest/pull/5272  ?

**Comment by sfan5** [comment]
No, the point of this PR is that you can use system-wide Android fonts instead of duplicating them in the copied assets.

**Comment by sofar** [comment]
This patch is very outdated. We now ship different fonts, and those are not a duplicate of the system fonts. Fixing it up wouldn't make much sense.

**Pull request closed without merging**